src/officehours_api/consumers.py

```diff
@@ -299,7 +239,8 @@ def trigger_user_deleted(sender, instance: User, **kwargs):
 @receiver(post_save, sender=Profile)
 @receiver(post_delete, sender=Profile)
 def trigger_user_update_for_profile(sender, instance: Profile, **kwargs):
-    transaction.on_commit(lambda: send_user_update(instance.user.id))
+    user_id = instance.user.id
+    transaction.on_commit(lambda: send_user_update(user_id))
```

Comment:
Comment might be useful here to explain why the assignment is needed. Also, could you create an issue for the user deletion errors and link it to this PR? I could too if you want, but I think you understand better what the issue is/was.